sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -279,10 +278,12 @@ private static void init(final @NotNull SentryOptions options, final boolean glo
         return;
       }
 
+      final boolean globalHubModeToUse =
```

Comment:
(l) getting an IDE warning here for a possible null-pointer due to options.isGlobalHubMode changing between checks. We could extract a `final Boolean globalHubModeFromOptions = options.isGlobalHubMode()` and use that in the ternary operator

Reply:
Ah yeah, forgot to do that, will update